barretenberg/cpp/src/barretenberg/polynomials/polynomial.hpp

```diff
@@ -149,6 +149,16 @@ template <typename Fr> class Polynomial {
         return std::span{ coefficients_.get() + 1, size_ };
     }
 
+    /**
+     * @brief Set self to the right shift of input coefficients
+     * @details Set the size of self to match the input then set coefficients equal to right shift of input, assuming
+     * last shift-size many inputs are zero.
```

Comment:
What does "assuming last shift-size many inputs are zero" mean?

Reply:
Improved the wording of this comment